ui/apps/platform/src/Containers/ComplianceEnhanced/ClusterCompliance/Coverage/Clusters/ClusterDetailsHeader.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Flex, Label, LabelGroup, Skeleton, Title } from '@patternfly/react-core';
+
+import { ComplianceClusterOverallStats } from 'services/ComplianceEnhancedService';
+
+import {
+    getPassAndTotalCount,
+    calculateCompliancePercentage,
+    getComplianceLabelGroupColor,
+} from '../compliance.coverage.utils';
+
+export type ClusterDetailsHeaderProps = {
+    clusterStats: ComplianceClusterOverallStats | undefined;
+};
+
+function ClusterDetailsHeader({ clusterStats }: ClusterDetailsHeaderProps) {
+    let passCount;
+    let totalCount;
+    let compliancePercentage;
+
+    if (clusterStats) {
+        ({ passCount, totalCount } = getPassAndTotalCount(clusterStats.checkStats));
+        compliancePercentage = calculateCompliancePercentage(passCount, totalCount);
+    }
+
+    return (
+        <>
+            <Flex direction={{ default: 'column' }} alignItems={{ default: 'alignItemsFlexStart' }}>
+                <Title headingLevel="h1" className="pf-u-mb-sm">
```

Comment:
Can you see if a prop like `spaceItems={{ default: 'spaceItemsSm' }}` in `Flex` element has the same effect as the utility classname? Two benefits to lift it up:
* Consistent spacing if you need to add a third child.
* Absence of unwanted spacing if you need to render the second child conditionally.

Reply:
This extra space isn't even necessary I'll go ahead and remove it